app/controllers/BaseController.scala

```diff
@@ -29,14 +30,22 @@ import uk.gov.hmrc.play.http.HeaderCarrier
 
 import scala.concurrent.Future
 
-case class FaststreamConfig(newAccountsEnabled: Boolean, applicationsSubmitEnabled: Boolean)
+case class ApplicationRouteConfig(newAccountsStarted: Boolean, newAccountsEnabled: Boolean, applicationsSubmitEnabled: Boolean)
```

Comment:
I am not sure that the naming here is really clear. `newAccountsStarted` and `newAccountsEnabled` are very similar. It seems that `newAccountsEnabled` is really `existingAccountsEnabled`